cmd/minishift/cmd/start.go

```diff
@@ -160,6 +160,15 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	if !isRestart {
+		// Run the default addons if MINISHIFT_HOME dir does not exist or empty
```

Comment:
No, I mean the changes in start.go in general. We are doing the unpacking in root.go. We don't have to touch start.go (I think)